src/test/groovy/org/prebid/server/functional/tests/pg/TokenSpec.groovy

```diff
@@ -236,4 +237,35 @@ class TokenSpec extends BasePgSpec {
                 [plansResponse.lineItems[0].lineItemId] as Set
         assert !secondAuctionResponse.ext?.debug?.pgmetrics?.sentToBidder
     }
+
+    def "PBS should start using line item in auction when pacing header is provided"() {
```

Comment:
"PBS should ignore line item pacing when ignore pacing header is present in the request"